src/app/pages/ArticlePage/index.test.jsx

```diff
@@ -274,11 +274,11 @@ it('should render a news article correctly', async () => {
 });
 
 it('should render a rtl article (persian) with most read correctly', async () => {
-  fetch.mockResponse(JSON.stringify(persianMostReadData));
-
   const { container } = render(
     <Context service="persian">
-      <ArticlePage pageData={articleDataPersian} />
+      <ArticlePage
+        pageData={{ ...articleDataPersian, mostRead: persianMostReadData }}
```

Comment:
I am wondering if it might it be easier to make sure that the `articleDataPersian` object includes `mostRead` so that we don't have to add it here like this?

Reply:
Ignore me, I just remembered why we had to do it this way. Let's leave it 👍